tests/unit/s2n_early_data_io_api_test.c

```diff
@@ -346,8 +347,12 @@ int main(int argc, char **argv)
             EXPECT_FAILURE_WITH_ERRNO(s2n_negotiate_test_server_and_client(server_conn, client_conn),
                                 S2N_ERR_BAD_MESSAGE);
 
+            /* Read the remaining early data properly */
+            server_conn->closed = false;
+            client_conn->closed = false;
```

Comment:
This seems like an issue no? If we can't continue to read early data without changing the conn->closed value? Is the user expected to change these values?

Reply:
I added this comment:
```
            /* Pretend we didn't test the above error condition.
             * The S2N_ERR_BAD_MESSAGE error triggered S2N to close the connection. */
```

Basically, in a production environment the connection failed after that S2N_ERR_BAD_MESSAGE. The test just wants to explore the possibility that it didn't.